src/oss/confluent.sh

```diff
@@ -95,6 +95,14 @@ declare -a commands=(
     "config"
 )
 
+declare -a dynamic_cli_commands=(
+    "sr-acl"
```

Comment:
my 2c: 

1. I don't like hyphens for name-spacing, we should avoid that even if it makes the implementation more difficult.

2. I think there should be component level name-spacing for everything, right from the start:
  1. it's more descriptive of what's happening (-> a better user experience + less potential for confusion) even if it's more verbose. e.g. i think `confluent connect load ...` is better than `confluent load ...` and `confluent connect status [connector name]` is better than `confluent status [connector name]`. 
  2. future-proofing. What if we want to load something else in the future? what if it's more important than a connector? what if I want to name a connector 'kafka' - `confluent status kafka`

3. I also don't like the ordering of the arguments. I think it should be e.g. `confluent kafka start` not `confluent start kafka`. In related news I also don't like writing dates as: mm/dd/yyyy. We have an obvious hierarchy, it would be more straightforward if the CLI matched this.